CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 in development
 --------------
 
+Changed:
```

Comment:
Small more of a personal preference thing - maybe get rid of semi-colon?

Besides that, LGTM